sentinel/lib/lineage.js

```diff
@@ -22,7 +22,7 @@ const mergeLineages = (lineage, patientLineage) => {
   const lineageContacts = [],
         contactsToFetch = [];
   contactIds.forEach(id => {
-    const contact = lineage.find(d => d && d._id === id);
+    const contact = lineages.find(d => d && d._id === id);
```

Comment:
\*cough\* types \*cough\*

:shipit:

Reply:
`lineage` and `lineages` have the same type... both arrays. I think "*cough* naming *cough*" is valid though.